lib/node_modules/@stdlib/repl/test/integration/fixtures/syntax-highlighting/control_keywords.json

```diff
@@ -0,0 +1,7 @@
+{
+  "expression": "async function foo(a) { if (a) { return a; } else { try { switch(a) { case 0: return await Promise.resolve(a); } } catch (e) { return e; } } }",
```

Comment:
This is tricky, as this assumes test environments supporting `async`. But we support environments only supporting ES5. Should be possible to add support for indicating whether environment detection needs to be performed prior to running a test. We do this in other tests (e.g., iterators and checking whether the env supports Symbols).

Reply:
@kgryte We are just checking the syntax highlighter and not actually executing a script in this case though.

Reply:
Ah. Okay. That works then. Thanks for clarifying!